store/plugin-commands-store-search/README.md

```diff
@@ -0,0 +1,15 @@
+# @pnpm/plugin-commands-store-search
```

Comment:
I think `plugin-commands-store-inspecting` will be a better name for this package.

Reply:
okay